xkcd/main.go

```diff
@@ -155,8 +159,8 @@ func Show(app fyne.App) {
 	w := app.NewWindow("XKCD Viewer")
 
 	form := x.NewForm(w)
-	x.image = &canvas.Image{}
```

Comment:
This portion has been implemented. Do you want to update to apply the other code or close and open a new one?